lerna.json

```diff
@@ -2,5 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "independent"
+  "version": "0.2.18",
+  "npmClient": "yarn",
+  "useWorkspaces": true
```

Comment:
can't we use yarn workspaces directly instead of lerna?

Reply:
yarn is for installing packages, lerna is for bumping versions

Reply:
Can't we use [`yarn version`](https://classic.yarnpkg.com/en/docs/cli/version/)? 
I don't think it is necessary to have lerna installed for it tbh

Reply:
Cannot figure this out for the moment, yarn alone doesn't work for me. Let's create an issue for this